crates/teloxide/examples/deep_linking.rs

```diff
@@ -0,0 +1,133 @@
+use dptree::{case, deps};
+use teloxide::{
+    dispatching::dialogue::{self, InMemStorage},
+    macros::BotCommands,
+    prelude::*,
+    types::Me,
+};
+
+pub type MyDialogue = Dialogue<State, InMemStorage<State>>;
+pub type HandlerResult = Result<(), Box<dyn std::error::Error + Send + Sync>>;
+
+#[derive(Clone, PartialEq, Debug, Default)]
+pub enum State {
+    #[default]
+    Start,
+    WriteToSomeone {
+        id: i64,
```

Comment:
Maybe to wrap the `id` into the `ChatId` here?

Reply:
Yup, will fix